testsuite/MDAnalysisTests/test_distances.py

```diff
@@ -63,12 +67,20 @@ def mindist(a, b, box):
             return np.linalg.norm(x - np.rint(x / box) * box)
 
         ref = mindist(a, b, box)
-        val = MDAnalysis.lib.distances.distance_array(np.array([a]), np.array([b]), box)[0, 0]
+        val = MDAnalysis.lib.distances.distance_array(np.array([a]), np.array([b]),
+                                                      box=box, mode=self.mode)[0, 0]
 
-        assert_almost_equal(val, ref, decimal=6, err_msg="Issue 151 not correct (PBC in distance array)")
+        assert_almost_equal(val, ref, decimal=6,
+                            err_msg="Issue 151 not correct (PBC in distance array)")
 
+class TestDistanceArray_Serial(_TestDistanceArray):
+    mode = "serial"
 
-class TestDistanceArrayDCD(TestCase):
+class TestDistanceArray_OpenMP(_TestDistanceArray):
```

Comment:
That's actually a really good point, currently we have no way of knowing how the Cython was compiled from the module.

Some attribute like `MDAnalysis.lib._distances_openmp.USED_OPENMP` would save us some headaches in the future.

I'll have a look at how we can do this.  Not a showstopper though.